library/options.inc.php

```diff
@@ -1893,7 +2070,7 @@ function generate_print_field($frow, $currvalue)
         }
 
         echo "</table>";
-    } elseif ($data_type == 27 || $data_type == 1 || $data_type == 26 || $data_type == 46) {
+    } elseif ($data_type == 27 || $data_type == 46) { // Removed: || $data_type == 1 || $data_type == 26 || $data_type == 33
```

Comment:
i think 46 should also go since now captured above (i always wondered why that `false` thing was there)

Reply:
false was there because I anticipated re-using the code. :)